ticker/ticker.go

```diff
@@ -426,6 +459,16 @@ func (t *Ticker) Close() error {
 	return t.Conn.WriteMessage(websocket.CloseMessage, websocket.FormatCloseMessage(websocket.CloseNormalClosure, ""))
 }
 
+// Stop the ticker instance and all the goroutines it has spawned.
+func (t *Ticker) Stop() error {
+	if t.cancel == nil {
+		return fmt.Errorf("ticker is not running")
```

Comment:
Why do we have to raise error if its not running? in that case `Stop()` doesn't need to return anything.